changelog/_unreleased/2020-09-17-fix-product-seo-url-handling-for-translations-in-administration.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Fix product SEO url handling for translations in administration
+issue: NEXT-9809
+author: Timo Helmke
+author_email: t.helmke@kellerkinder.de 
+author_github: @t2oh4e
+___
+# Administration
+* Fixed the handling of SEO urls for products with languages other than the default language and new sales channels.
```

Comment:
Please uses only of the following keywords: Added, Removed, Changed, Deprecated.
It should describe what you changed in the code.

E.g. 
```
* Added properties `foreignKey`, `pathInfo` to `sw-url-field`, `routeName` to  `sw-seo-url` component
```

Reply:
Since the fix included like 3 changes I changed the changelog to display those better.